tern/classes/package.py

```diff
@@ -90,11 +100,33 @@ def checksum(self):
     def checksum(self, checksum):
         self.__checksum = checksum
 
+    def get_file_paths(self):
+        """Return a list of paths of all the files in a package"""
+        return list(map(lambda f: f.path, self.__files))
```

Comment:
I think this is a bit overkill and hurts readability, particularly if you are casting the returned object multiple times. How about using list comprehension if you want a one-liner: `return [f.path for f in self.__files]`

Reply:
list comprehension is way better. Thanks